modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -393,23 +393,23 @@ interface ObjectDefinitionNodeData
 	selected: boolean;
 }
 
-interface PickListItem {
+interface ListTypeDefinition {
+	actions: Actions;
 	externalReferenceCode: string;
 	id: number;
 	key: string;
+	listTypeEntries: ListTypeEntry[];
```

Comment:
@MarinhoFeliphe @igor-franca ListTypeEntry is used here, that's why it was declared before to respect precedence order.

Reply:
This interface has been renamed in api.ts and index.d.ts

Reply:
@MarinhoFeliphe , can you make this modification until we align on the sorting pattern used?

Reply:
Sure!

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/2571/commits/0bd111742344a20b4e223a0e16fe815fbc59102b)